Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

```diff
@@ -50,6 +51,15 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
+            // check for ternary operator
+            $endParenthesisIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+            $nextNonWhiteSpace = $tokens->getNextMeaningfulToken($endParenthesisIndex);
+            if (!empty($nextNonWhiteSpace) && $tokens[$nextNonWhiteSpace]->equals('?')) {
+                if ($tokens[$lastTokenIndex]->isGivenKind($languageConstructionTokens)) {
```

Comment:
It's not a matter of demand, just a bit performance and consistency.
then why not `if 1... if 2... if 3... ` ? :)

One could still write it readable, sth like:
```php
if (
    1 &&
    2 &&
    3
) {
    sth();
}
```